src/Raven.Server/Documents/Queries/IndexQueryServerSide.cs

```diff
@@ -14,6 +14,16 @@ public class IndexQueryServerSide : IndexQuery<BlittableJsonReaderObject>
         [JsonDeserializationIgnore]
         public QueryMetadata Metadata { get; private set; }
 
+        /// <summary>
+        /// Gets or sets the cutoff etag.
+        /// <para>Cutoff etag is used to check if the index has already process a document with the given</para>
+        /// <para>etag. Unlike Cutoff, which uses dates and is susceptible to clock synchronization issues between</para>
```

Comment:
Do we even _have_ `Cutoff` anymore?

Reply:
No need for this indeed. https://github.com/ravendb/ravendb/pull/4613